### PR TITLE
Distribute hybrid driver.

### PR DESCRIPTION
**Pull request by sriharshakandala**
Distribute hybrid driver.

### CODE REVIEW

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try-

**Comment by sriharshakandala** [comment]
bors try

File: examples/Manifest.toml [hunk 1]
```diff
@@ -143,7 +143,7 @@ version = "0.1.1"
 
 [[deps.ClimaCore]]
 deps = ["Adapt", "BlockArrays", "CUDA", "ClimaComms", "CubedSphere", "DiffEqBase", "DocStringExtensions", "ForwardDiff", "GaussQuadrature", "IntervalSets", "LinearAlgebra", "RecursiveArrayTools", "RootSolvers", "Rotations", "SparseArrays", "Static", "StaticArrays", "Statistics", "UnPack"]
-path = ".."
+path = "/Users/sriharshakandala/work/ClimaCore.jl"
```
Comment (dennisYatunin) [inline examples/Manifest.toml:146]: Probably should not be committed

**Review by dennisYatunin — APPROVED** [review]

**Comment by jiahe23** [comment]
Thanks for the efforts, @sriharshakandala ! Could you modify the run script I shared in the dycore channel into a template if we want to run the distributed simulation on central?

**Comment by sriharshakandala** [comment]
bors try-

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try

**Pull request merged**